src/components/OnboardingForm.tsx

```diff
@@ -0,0 +1,163 @@
+"use client";
+export default function OnboardingForm() {
+  return (
+    <div className="absolute h-screen w-screen z-10 top-0 left-0 bg-slate-800 overflow-auto">
+      <form className="bg-[#5d417A] w-full md:w-9/12 mx-auto my-10 p-8 rounded-lg">
```

Comment:
Wow, why use this color? `bg-[#5d417A]`

This was part of the form template?

Reply:
We picked this color from [figma](https://www.figma.com/file/NRnn6iLgO0oMXgZm1fPPPH/webapp?type=design&t=ygye4KG6Nk3OplBf-0)

Maybe we should create CSS/Tailwind vars for these colors

Reply:
I'm gonna commit this with my next PR.

Reply:
Agree! Nice suggestion Nicolas!